examples/python3/Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM python:3-slim AS build-env
+FROM python:3.6-stretch AS build-env
```

Comment:
Why pin specifically 3.6?  And why `-slim` &rarr; `-stretch`?

Reply:
I will have to check but distroless is still built off of stretch right. I was attempting to give some context here as it is only an example. Happy to make changes in the future.